std/typelist.d

```diff
@@ -261,26 +261,34 @@ unittest {
     static assert(is(Filter!(IsPointer) == TypeTuple!()));
 }
 
+/**
+ * Folds (reduces) the $(D TypeList) $(D_PARAM List) using the function $(D_PARAM F), 
+ * and the initial value $(D_PARAM Init).
+ */
 template FoldRight(alias F, alias Init, alias List)
 {
     static if (List.isEmpty)
-        alias Init FoldRight;
+        alias FoldRight = Init;
     else
-        alias F!(List.head, FoldRight!(F, Init, List.tail)) FoldRight;
+        alias FoldRight = F!(List.head, FoldRight!(F, Init, List.tail));
 }
 
-template FoldRight(alias F, int Init, alias List)
```

Comment:
This line actually is different from line 264. `alias` vs `int`. I'm not sure if that means it's necessary though.